reps/2022-08-31-actor-affinity-apis.md

```diff
@@ -0,0 +1,538 @@
+
+## Summary
+### General Motivation
+
+Provides a set of lightweight actor affinity and anti-affinity scheduling interfaces.
+Replacing the heavy PG interface to implement collocate or spread actors.
+
+* Affinity
+  * Co-locate the actors in the same node.
+  * Co-locate the actors in the same batch of nodes, like nodes in the same zones
+* Anti-affinity
+  * Spread the actors of a service across nodes and/or availability zones, e.g. to reduce correlated failures.
```

Comment:
How is it possible to spread across AZs using this proposal?

Reply:
1. First use custom resource or other methods to schedule an actor to the specified AZs.
2. Then ohters actors just use actor-affinity feature to affinity with the actor corresponding to AZs.